tests/tabular_output/test_preprocessors.py

```diff
@@ -85,3 +90,44 @@ def test_quote_whitespaces_non_spaces():
                 ['h1', 'h2'])
 
     assert expected == quote_whitespaces(data, headers)
+
+
+@pytest.mark.skipif(not HAS_PYGMENTS, reason='requires the Pygments library')
+def test_style_output_no_styles():
+    """Test that *style_output()* does not style without styles."""
+    headers = ['h1', 'h2']
+    data = [['1', '2'], ['a', 'b']]
+
+    assert (data, headers) == style_output(data, headers)
+
+
+@pytest.mark.skipif(HAS_PYGMENTS, reason='requires no Pygments library')
```

Comment:
Haha, yeah, this isn't very clear. It's actually testing that things work properly when you don't have Pygments installed. So:

- Two `pytest.mark.skipif` tests that require Pygments to be installed.
- One `pytest.mark.skipif` test that requires Pygments be missing.

I'll reword the `reason` so it's more clear.